cmake/XRootDSummary.cmake

```diff
@@ -12,7 +12,7 @@ component_status( TESTS     BUILD_TESTS       CPPUNIT_FOUND )
 component_status( HTTP      BUILD_HTTP        OPENSSL_FOUND )
 component_status( TPC       BUILD_TPC         CURL_FOUND )
 component_status( MACAROONS BUILD_MACAROONS   MACAROONS_FOUND )
-component_status( PYTHON    BUILD_PYTHON      PYTHON_FOUND )
+component_status( PYTHON    BUILD_PYTHON      Python_FOUND )
```

Comment:
This should probably match the other check, i.e.
```
Python_Interpreter_FOUND AND Python_Development_FOUND
```

Reply:
@amadio I updated this from `Python_FOUND` to `Python_Interpreter_FOUND AND Python_Development_FOUND` as you suggested though note from https://cmake.org/cmake/help/v3.12/module/FindPython.html#result-variables that `Python_FOUND` is a boolean of if 

> System has the Python requested components

and so would be more robust to changes in the associated `find_package` command.